engine/inc/irx/irx_loader.hpp

```diff
@@ -27,6 +27,7 @@ class IrxLoader {
   void loadSio2man(const bool& verbose);
   void loadPadman(const bool& verbose);
   void loadLibsd(const bool& verbose);
+  void loadPOSIX(const bool& verbose);

```

Comment:
Maybe `loadIO()`?. I think that POSIX name too too generic.

Reply:
Good name, I'll refactor this.
The DEMO was tested in the PCSX2 but no in the real hardware.
I'll certify it's working on both of them beforo update the PR

Reply:
> Good name, I'll refactor this. The DEMO was tested in the PCSX2 but no in the real hardware. I'll certify it's working on both of them beforo update the PR

I totally forgot about these pending checks. I'll test in the real hardware and comeback with the results

Reply:
neat

Reply:
this should be called load extended IO or something...

The POSIX trick is caused by the libcglue library.

fileXio and iomanX are just eXtended I/O modules, that co-exist with their stock versions (`rom0:IOMAN`, `rom0:FILEIO`)

Reply:
@Wellinator over here ⬆️

Reply:
Hey guys! Thx for the suggestions. 
Gonna finish it rn.

Reply:
I have a new branch based on yours, i think you don't need it.

Reply:
I'm testing in the real hardware